modules/terminations.stanza

```diff
@@ -64,4 +80,126 @@ public defn terminate-usb-2 (bus:JITXObject) :
     res-strap(terminated-usb-2.data.N, bus.data.N, 22.0)
     res-strap(terminated-usb-2.data.P, bus.data.P, 22.0)
     ; Return the terminated bus
-    terminated-usb-2
\ No newline at end of file
+    terminated-usb-2
+
+pcb-module termination (method:LVDS-Termination-Method) :
```

Comment:
I would give this module a more descriptive name `lvds-termination` or something. Given the new approach there will probably be a few definitions like this in this file